mslib/mscolab/file_manager.py

```diff
@@ -250,6 +258,71 @@ def modify_user(self, user, attribute=None, value=None, action=None):
             db.session.commit()
         return True
 
+    def delete_user_profile_image(self, image_to_be_deleted):
+        '''
+        This function is called when deleting account or updating the profile picture
+        '''
+        upload_folder = mscolab_settings.UPLOAD_FOLDER
+        with fs.open_fs(upload_folder) as profile_fs:
```

Comment:
This uses UPLOAD_FOLDER without first replacing bashlashes with slashes on Windows.